permission-management/clone-object-permissions-wizard.md

```diff
@@ -0,0 +1,44 @@
+**Supported source SharePoint Object selection**: all object levels from site collection root site to list item.  
+**Supported target SharePoint Object selection**: all object levels from site collection root site to list item.
+
+1.  Navigate to Permissions Explorer and select the source object in the object tree. Select the **Clone Object Permissions** options in the **Manage** ribbon.
+
+2. In the **Target selection** step select the target SharePoint object. 
+
+3. Depending on the previous source and target object selection, the **Configuration** step offers a couple of options. 
+
+    * **Source and target objects are a part of the same site collection.**
+
+        The **Keep existing target object permissions** option is available. If checked, all of the target object permissions are preserved and new permissions are added to existing groups or principals. If not checked, all target object permissions are removed and the source permissions are recreated.
+
+        Options **Create SharePoint groups** and **Create missing permission levels** are disabled since they are only applicable in case of different site collections.
+
+        If the target object inheritance is not broken, the following message is displayed: **The target securable object currently inherits permissions from its parent. Permission inheritance will be broken in order to apply new role assignments**.
+
+    * **Source and target objects are a part of different site collections.** 
+
+        The **Create SharePoint groups** option is available. If checked, missing SharePoint groups are recreated on the target site collection with the same members as in the source group. If unchecked, no new groups are created.
+
+        The **Create missing permission levels** option is also available. If checked, missing permission levels are recreated on the target site collection with the same base permissions as in the source permission levels. If unchecked, the permission levels are not created and therefore can not be assigned to groups or principals.
+
+        **Please note!** If the source site collection and the target site collection have SharePoint groups with the same name and different members, and the Keep existing target object permissions option is checked, existing members in the target group are preserved and new members from the source group are added.
+        If the Keep existing target object permissions option is not checked, the target group members are removed and source group members recreated. The same rules apply to the permission levels.  
+
+4. The **Preview** step shows the changes that will occur in the final step. 
+
+5. In the **Apply Changes** step, all changes are displayed as they are applied. It is possible to save the log for later reference by clicking the **Save Log** button. The changes are also saved in the [SPDocPerm event log](#internal/permission-management/spdockit-permission-management-event-log).
+
```

Comment:
ispred imena opcija **nejde** The? npr. The **Create missing permission levels** option
The **Create SharePoint groups**
The **Keep existing target object permissions**

Reply:
u KB wordu 'Pravilnik za pisanje dokumentacije' piše:

Imena proizvoda , komandi, programa, itd. pišu se velikim početnim slovom...
Isto tako u večini slučajeva te iste riječi imaju i član na početku (a,the).

Reply:
hmm ne znam, ne zvuči mi dobro. fixaj sve ostalo ovo ostavi za kraj. pa ćemo vidjet